### DISCUSSION TITLE
Stuck in bootloop

### DISCUSSION THREAD

**Discussion by MrNIMJE** [phhusson/treble_experimentations#1818] [Q&A]
Hey I am on redmi note 9 pro curtana and I am stuck in bootloop on flashing gsi can someone help me pls ?

**Comment by 00p513-dev** [comment]
Try with latest MIUI vendor. If you still have issues please provide a logcat.

To take a logcat run: `adb logcat -d > logcat.txt` and send the logcat.txt here :)

**Comment by amrithmmh** [comment]
did you decrypt/ disable verify on boot?
did you resize the system partition?
